test/reduce-sdk-e2e/testdata/reduce-stream/reduce-stream-go.yaml

```diff
@@ -0,0 +1,45 @@
+apiVersion: numaflow.numaproj.io/v1alpha1
+kind: Pipeline
+metadata:
+  name: reduce-stream-go
+spec:
+  vertices:
+    - name: in
+      source:
+        http: {}
+    - name: atoi
+      scale:
+        min: 1
+      udf:
+        container:
+          # Tell the input number is even or odd, see https://github.com/numaproj/numaflow-go/tree/main/pkg/mapper/examples/even_odd
+          image: quay.io/numaio/numaflow-go/map-even-odd:v0.6.0
+    - name: compute-sum
+      partitions: 2
+      udf:
+        container:
+          # compute the sum, see https://github.com/numaproj/numaflow-go/tree/main/pkg/reducestreamer/examples/sum
+          image: quay.io/numaio/numaflow-go/reduce-stream-sum:v0.6.0
+        groupBy:
+          window:
+            fixed:
+              length: 60s
+              streaming: true
+          keyed: true
```

Comment:
Could we use a longer window, like 10-20 minutes, and check if the responses are coming in the sink even before the window gets closed?

Reply:
Sure will do.